doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,494 @@
+    >>> plt.imshow(img)
+
+.. plot:: user/plot_face.py
+    :align: center
+    :include-source: 0
+
+.. note::
+
+   If you are executing the commands above in the IPython shell, it might be necessary to use the
+   command ``plt.show()`` to show the image window. 
+		     
+**Shape, axis and array properties**
+
+Note that, in linear algebra, the dimension of a vector refers to the number of entries in an
+array. In NumPy, it instead defines the number of axes. For example, a 1D array is a vector
+such as ``[1, 2, 3]``, a 2D array is a matrix, and so forth.
+
+First, let's check for the shape of the data in our array. Since this image is
+two-dimensional (the pixels in the image form a rectangle), we might expect a
+two-dimensional array to represent it (a matrix). However, using the ``shape``
+property of this NumPy array gives us a different result::
+
+    >>> img.shape
+    (768, 1024, 3)
+
+The output is a `tuple` with three elements, which means that this is a
+three-dimensional array. In fact, since this is a color image, and we have used
+the ``imread`` function to read it, the data is organized in three 2D arrays,
+representing color channels (in this case, red, green and blue - RGB). You can
+see this by looking at the shape above: it indicates that we have an array of 3
+matrices, each having shape 768x1024.
+
+Furthermore, using the ``ndim`` property of this array, we can see that
+
+::
+
+    >>> img.ndim
+    3
+
+NumPy refers to each dimension as an `axis`. Because of how ``imread``
+works, the *first index in the 3rd axis* is the red pixel data for our image. We
+can access this by using the syntax
+
+::
+
+    >>> img[:, :, 0]
+    array([[121, 138, 153, ..., 119, 131, 139],
+           [ 89, 110, 130, ..., 118, 134, 146],
+           [ 73,  94, 115, ..., 117, 133, 144],
+           ...,
+           [ 87,  94, 107, ..., 120, 119, 119],
+           [ 85,  95, 112, ..., 121, 120, 120],
+           [ 85,  97, 111, ..., 120, 119, 118]], dtype=uint8)
+
+From the output above, we can see that every value in ``img[:,:,0]`` is an
+integer value between 0 and 255, representing the level of red in each
+corresponding image pixel (keep in mind that this might be different if you
+use your own image instead of `scipy.misc.face`).
+
+As expected, this is a 768x1024 matrix::
+
+    >>> img[:, :, 0].shape
+    (768, 1024)
+
+Since we are going to perform linear algebra operations on this data, it might
+be more interesting to have real numbers between 0 and 1 in each entry of the
+matrices to represent the RGB values. We can do that by setting
+
+    >>> img_array = img / 255
+
+This operation, dividing an array by a scalar, works because of NumPy's `broadcasting`
+rules (see :ref:`array-broadcasting-in-numpy`). (Note that in real-world
+applications, it would be better to use, for example, the `img_as_float
+<https://scikit-image.org/docs/dev/api/skimage.util.html#skimage.util.img_as_float>`_
+utility function from ``scikit-image``).
```

Comment:
Also could be an intersphinx link if we add https://scikit-image.org/docs/dev, the link would be `skimage.img_as_float`